securedrop/journalist_templates/_sources_confirmation_final_modal.html

```diff
@@ -0,0 +1,19 @@
+<div id="{{ modal_data.modal_id }}" class="menu-modal-dialog">
+  <a href="#close" class="external"></a>
+  <div id="delete-confirm-menu-dialog">
+    <p>
+      {{ gettext('When the login account for a source is deleted:') }}
+      <ul>
+        <li>{{ gettext('The source will not be able to log in with their codename again') }}</li>
```

Comment:
The subsequent bullets end with a `.` but this one does not; all are complete sentences.

Reply:
:heavy_check_mark: